### ISSUE TITLE
Full report is broken

### ISSUE THREAD

**Issue by degloff** [ranaroussi/quantstats#95]
Full report seems to be broken.

````
bench = qs.utils.download_returns('SPY')
qs.reports.full(bench, benchmark=bench)
````

Produces error

```
~/opt/anaconda3/envs/test/lib/python3.8/site-packages/quantstats/_plotting/core.py in plot_histogram(returns, resample, bins, fontname, grayscale, title, kde, figsize, ylabel, subtitle, compounded, savefig, show)
    348     if subtitle:
    349         ax.set_title("\n%s - %s                   " % (
--> 350             returns.index.date[:1][0].strftime('%Y'),
    351             returns.index.date[-1:][0].strftime('%Y')
    352         ), fontsize=12, color='gray')

TypeError: 'method' object is not subscriptable
```

**Comment by ranaroussi** [comment]
Fixed in 0.0.32

**Comment by Cokenach** [comment]
Hi using 0.034, the report seems to be broken again.

`
bench = qs.utils.download_returns('SPY')
qs.reports.full(bench, benchmark=bench)
`

Produces this error:

`Performance Metrics
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-88-1d84dab83239> in <module>
----> 1 qs.reports.full(bench, benchmark=bench)

C:\ProgramData\Anaconda3\lib\site-packages\quantstats\reports.py in full(returns, benchmark, rf, grayscale, figsize, display, compounded, trading_year_days)
    238         iDisplay(metrics(returns=returns, benchmark=benchmark,
    239                          rf=rf, display=display, mode='full',
--> 240                          compounded=compounded))
    241         iDisplay(iHTML('<h4>5 Worst Drawdowns</h4>'))
    242         if dd_info.empty:

C:\ProgramData\Anaconda3\lib\site-packages\quantstats\reports.py in metrics(returns, benchmark, rf, display, mode, sep, compounded, trading_year_days, **kwargs)
    391     metrics['~~~~~~'] = blank
    392 
--> 393     metrics['Gain/Pain Ratio'] = _stats.gain_to_pain_ratio(df, rf)
    394     metrics['Gain/Pain (1M)'] = _stats.gain_to_pain_ratio(df, rf, "M")
    395     # if mode.lower() == 'full':

C:\ProgramData\Anaconda3\lib\site-packages\quantstats\stats.py in gain_to_pain_ratio(returns, rf, resolution)
    266     https://archive.is/wip/2rwFW
    267     """
--> 268     returns = _utils._prepare_returns(returns, rf).resample(resolution).sum()
    269     downside = abs(returns[returns < 0].sum())
    270     return returns.sum() / downside

C:\ProgramData\Anaconda3\lib\site-packages\pandas\core\generic.py in resample(self, rule, axis, closed, label, convention, kind, loffset, base, on, level)
   8099 
   8100         To replace the use of the deprecated `loffset` argument:
-> 8101 
   8102         >>> from pandas.tseries.frequencies import to_offset
   8103         >>> loffset = '19min'

C:\ProgramData\Anaconda3\lib\site-packages\pandas\core\resample.py in <module>
     14 
     15 from pandas._libs import lib
---> 16 from pandas._libs.tslibs import (
     17     BaseOffset,
     18     IncompatibleFrequency,

ImportError: cannot import name 'BaseOffset' from 'pandas._libs.tslibs' (C:\ProgramData\Anaconda3\lib\site-packages\pandas\_libs\tslibs\__init__.py)
`
